test/bats/test.bats

```diff
@@ -3,7 +3,7 @@ load test_helper
 
 @test "Can connect to userdb PostgreSQL" {
 	sleep 1
-	run pg_isready -U postgres -h userdb;
+	run pg_isready -U postgres -h userdb-postgres;
```

Comment:
You changed the host but not the docker run script.

Reply:
the docker run script was already set to this. The tests were always failing.